cg/services/order_validation_service/models/order_with_samples.py

```diff
@@ -1,11 +1,10 @@
-from cg.services.order_validation_service.models.aliases import NonHumanSample
 from cg.services.order_validation_service.models.order import Order
 from cg.services.order_validation_service.models.sample import Sample
 
 
 class OrderWithNonHumanSamples(Order):
```

Comment:
Should this be renamed to `OrderWithSamples`?

Reply:
Maybe? At least the current naming is not correct, since we do have human orders without cases (such as fastq). Just wondering if OrderWithSamples will cause confusion since all orders have samples, it is just a matter of whether they have cases or not. OrderWithSanples should be better than the current naming though

Reply:
Ok but now I understand that this is not involved in the inheritance. Therefore the renaming is out of scope

Reply:
I do think we need to have a think about what the proposed change with how we type hint the samples will mean. See for example the rule `validate_organism_exists` - for all the samples which the "NonHumanSample" encompass (Mutant and Microsalt thus far), we will have a sample.organism attribute that we can access. If we change the type hint here to `Sample` instead of `NonHumanSample` there is no guarantee that there actually is an `organism`attribute.

I think I like your proposed change since Fastq orders will not be nonhuman, and maybe we should look into adding some order aliases to remedy the failing attribute accessing? So we could add `NonHumanOrderTypes = MicrosaltOrder | MutantOrder` and change the type hint in `validate_organism_exists` to use that alias?